src/Translatable/Mapping/Driver/Yaml.php

```diff
@@ -80,7 +80,7 @@ protected function _loadMappingFile($file)
         return \Symfony\Component\Yaml\Yaml::parse(file_get_contents($file));
     }
 
-    private function buildFieldConfiguration($field, $fieldMapping, array &$config)
+    private function buildFieldConfiguration(string $field, $fieldMapping, array &$config): void
```

Comment:
```suggestion
    private function buildFieldConfiguration(string $field, array $fieldMapping, array &$config): void
```

based on 

https://github.com/doctrine-extensions/DoctrineExtensions/blob/477dca8ca53a418c04a46dad6c30253837a04312/src/Sluggable/Mapping/Driver/Yaml.php#L90

Reply:
But in this case, there is a check against `is_array()` for the argument 2. If the check is not satisfied, currently the function does nothing.

Reply:
yeah, sorry, what I meant is that those methods are called with the same context:

https://github.com/doctrine-extensions/DoctrineExtensions/blob/477dca8ca53a418c04a46dad6c30253837a04312/src/Sluggable/Mapping/Driver/Yaml.php#L54-L64

and

https://github.com/doctrine-extensions/DoctrineExtensions/blob/477dca8ca53a418c04a46dad6c30253837a04312/src/Translatable/Mapping/Driver/Yaml.php#L56-L66

So `$fieldMapping` should be always an array (if in one place is typed as `array`).